### ISSUE TITLE
Music library not being processed

### ISSUE THREAD

**Issue by jose1711** [YunoHost-Apps/jellyfin_ynh#95]
### Describe the bug

Wonder if it's just me but for some reason music-type library is always empty - things are just not being populated. Videos work fine.

### Context

- Hardware: *VPS bought online 

### Steps to reproduce

1. Throw a few mp3 files in
2. Create a new Music-type library
3. Force update manually
4. Watch how nothing happened - library is empty
5. Nothing in logs

**Comment by jose1711** [comment]
Good point. I placed them into `/home/yunohost.multimedia/share/Music`. As soon as I copy them to a newly created `/mp3` they become visible though. But this does not seem to be permission-related. As `jellyfin` user I can access files in both locations. Not really sure what makes the original location so special..

**Comment by tituspijean** [comment]
After investigation (https://jellyfin.org/docs/general/server/media/music.html), I would guess it's normal. Jellyfin expects the files to be in a subdirectory (by artist).

**Comment by Salamandar** [comment]
Oh, yeah, that makes perfect sense, actually ^^''

**Comment by jose1711** [comment]
Thank you for your suggestions - after taking a closer look I realized I made a mistake right at the beginning. My other movies library was set to `/home/yunohost.multimedia/share` instead of `/home/yunohost.multimedia/share/Video`. Apparently Jellyfin is unable to cope with situations where library folders overlap. All is good since I changed the location.

**Comment by Salamandar** [comment]
Nice, thank you for troubleshooting yourself the issue :) more users should do like you :)
